config/jobs/kubernetes/sig-scalability/sig-scalability-presubmit-jobs.yaml

```diff
@@ -16,6 +16,7 @@ presubmits:
       - args:
         - --root=/go/src
         - --repo=k8s.io/kubernetes=$(PULL_REFS)
+        - --repo=k8s.io/perf-tests=master
```

Comment:
Can you please split just non-kubemark presubmits to a separate PR? I would like to understand the reasoning behing 5 jobs for kubemarks first.

Reply:
#11171 created.

Reply:
I removed gce to #11171. Only kubemark presubmits left.